hta/common/trace_parser.py

```diff
@@ -428,3 +436,102 @@ def parse_trace_dataframe(
             f"Parser Memory usage peak = {(peak/1024/1024):.2f} MB, current = {(current/1024/1024):.2f} MB"
         )
     return meta, df, local_symbol_table
+
+
+# --- Trace metadata reader ---
+def parse_metadata_ijson(fh) -> MetaData:
```

Comment:
is the absence of a type annotation on `fh` intentional since `ijson.parse` can accept different "file-like" objects?

Reply:
Just me being lazy, I'll add it